health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/icons/ShapeFileIcon.js

```
@@ -0,0 +1,65 @@
+import React from "react";
+
+export const ShpFileIcon = () => (
+  <svg width="96" height="120" viewBox="0 0 96 120" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Biome

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)